x-pack/test/api_integration/apis/synthetics/add_monitor_project_private_location.ts

```diff
@@ -54,7 +54,7 @@ export default function ({ getService }: FtrProviderContext) {
       });
     });
 
-    it('project monitors - returns a failed monitor when creating integration fails', async () => {
+    it.skip('project monitors - returns a failed monitor when creating integration fails', async () => {
```

Comment:
Now that this bug is fixed, we can't use this for testing. Same below. Any suggestions on how to alternatively force an error?